packages/brokeneck-react/src/graphql.js

```diff
@@ -1,9 +1,12 @@
 export const LOAD_GROUP = (fields, userFields) => `
-query LoadGroup($id: String!) {
-  group(id: $id) {
+query LoadGroup($id: String!, $pageNumber: String, $pageSize: Int) {
+  group(id: $id, pageNumberUsers: $pageNumber, pageSizeUsers: $pageSize) {
```

Comment:
can you look into how more complex it would be to make the `users` field (line 5) accept these parameters instead? It is more conceptually correct to have them there since it's the users field you're paginating